pkg/crypto/hash/hash.go

```diff
@@ -67,7 +67,8 @@ func Hash160(data []byte) util.Uint160 {
 }
 
 // Checksum returns the checksum for a given piece of data
-// using sha256 twice as the hash algorithm.
+// using sha256 twice as the hash algorithm. It returns the
```

Comment:
Maybe we can reference DoubleSha256 directly here, it's defined in the same package.

Reply:
Yes, fixed.